src/java/fr/paris/lutece/plugins/forms/web/admin/FormResponseJspBean.java

```diff
@@ -614,10 +716,16 @@ private String getBackUrl( Form form, boolean bIsEndMessageDisplayed )
             {
                 url = new UrlItem( getViewUrl( VIEW_STEP ) );
             }
-            
+
             url.addParameter( FormsConstants.PARAMETER_ID_FORM, form.getId( ) );
 
             return url.getUrl( );
         }
     }
-}
\ No newline at end of file
+
+    private boolean isSessionLost( )
```

Comment:
No, the null value of these objects does not mean that the user has lost their session; a call to the init(HttpServletRequest request) method will reset all these objects to null.